authentication-saml/src/main/java/com/synopsys/integration/alert/authentication/saml/database/configuration/SAMLConfigurationEntity.java

```diff
@@ -64,19 +77,17 @@ public SAMLConfigurationEntity(
         this.entityBaseUrl = entityBaseUrl;
         this.wantAssertionsSigned = wantAssertionsSigned;
         this.roleAttributeMapping = roleAttributeMapping;
+        this.encryptionCertFilePath = encryptionCertFilePath;
+        this.signingCertFilePath = signingCertFilePath;
+        this.verificationCertFilePath = verificationCertFilePath;
+        this.metadataMode = metadataMode;
     }
 
-    public UUID getConfigurationId() {
-        return configurationId;
-    }
+    public UUID getConfigurationId() { return configurationId; }
 
-    public String getName() {
-        return name;
-    }
+    public String getName() { return name; }
 
-    public OffsetDateTime getCreatedAt() {
-        return createdAt;
-    }
+    public OffsetDateTime getCreatedAt() { return createdAt; }
```

Comment:
minor thing, but the returns should be on their own line

Reply:
Changed to multi-line